firmware/CMakeLists.txt

```diff
@@ -8,7 +8,11 @@ add_definitions(-DGIT_HASH="${COMMIT_ID}") # Usable in source code.
 add_definitions(-DUSBD_MANUFACTURER="Allen Institute")
 add_definitions(-DUSBD_PRODUCT="Environment Sensor")
 
-include(${PICO_SDK_PATH}/pico_sdk_init.cmake)
```

Comment:
Thanks for doing the work to make the build setup windows compatible.

Reply:
Ok I had a closer look at this, and it's not a cross-platform fix, but it's a step in that direction.

I just tweaked this and put it in a branch. Incoming PR to this branch!